test/testcafe/framework/page-objects/IdentityPageObject.js

```diff
@@ -189,4 +190,8 @@ export default class IdentityPageObject extends BasePageObject {
   getCustomIdpButtonLabel(index) {
     return Selector(CUSTOM_IDP_BUTTON).nth(index).textContent;
   }
+
+  async hasBackToSignInLink() {
```

Comment:
there is `goBackLinkExists` exists in `BasePageObject`.. probably you can reuse that.

Reply:
good catch! I didn't realize this existed already.